tasks/hyrax_dev.rake

```diff
@@ -34,6 +34,56 @@ task :i18n_sorter do
   end
 end
 
+# rubocop:disable Metrics/BlockLength
+
+Rake::Task["engine_cart:create_test_rails_app"].clear
+
+namespace :engine_cart do
+  task create_test_rails_app: [:setup] do
+    require 'tmpdir'
+    require 'fileutils'
+    Dir.mktmpdir do |dir|
+      # Fork into a new process to avoid polluting the current one with the partial Rails environment ...
+      pid = fork do
+        Dir.chdir dir do
+          require 'rails/generators'
+          require 'rails/generators/rails/app/app_generator'
+
+          # Clear out our current bundle so that the rails generator can run with it's own bundle
+          backup_gemfile = ENV['BUNDLE_GEMFILE']
+          ENV.delete('BUNDLE_GEMFILE')
+
+          # Using the Rails generator directly, instead of shelling out, to
+          # ensure we use the right version of Rails.
+          Rails::Generators::AppGenerator.start([
+            'internal',
+            '--skip-git',
+            '--skip-keeps',
+            '--skip_spring',
+            '--skip-bootsnap',
+            '--skip-listen',
+            '--skip-test',
+            '--skip-javascript',
```

Comment:
I added an override of the `engine_cart:create_test_rails_app` primarily to enforce skipping javascript in the generation of a new Rails app by default. In Rails 6, unless you use `--skip-javascript`, the `webpacker` gem is installed and leads to a variety of changes and errors on the app.

Reply:
Instead of overriding engine_cart, can we instead update relevant documentation and the CI config to insert `--skip-javascript` using the ENGINE_CART_RAILS_OPTIONS environment variable?

Reply:
@dlpierce I added the change you recommended, and I deleted the `engine_cart.yml` file, because it overrides the options set in the circleci config, and currently has a bug that does not allow for adding multiple options. Please refer to this thread for more details: https://github.com/cbeer/engine_cart/pull/102